src/consensus/pbft_manager.hpp

```diff
@@ -145,8 +145,7 @@ class PbftManager {
   std::shared_ptr<VoteManager> vote_mgr_ = nullptr;
   std::shared_ptr<PbftChain> pbft_chain_ = nullptr;
   std::shared_ptr<DagManager> dag_mgr_ = nullptr;
-  std::shared_ptr<Network> network_ = nullptr;
-  std::shared_ptr<TaraxaCapability> capability_ = nullptr;
+  util::WeakRef<Network> network_;
   std::shared_ptr<DagBlockManager> dag_blk_mgr_;
```

Comment:
Why not initialize these as well to nullptr?

Reply:
1) That `= nullptr` stuff is redundant and should be cleaned up 
2) WeakRef, which I wrote, is not like std library smart pointers. I didn't create conversion rules/constructors for pointers, including nullptr.  @justinsnapp